src/main/resources/META-INF/openapi.yml

```diff
@@ -582,6 +575,8 @@ components:
           type: string
         modelType:
           type: string
+        servingRuntimeType:
+          type: string
```

Comment:
Do you also want to include temperature and max tokens here? Or are those intended solely for internal use?